clients/web/src/views/widgets/UploadWidget.js

```diff
@@ -176,7 +176,15 @@ var UploadWidget = View.extend({
         this.$('.g-drop-zone')
             .removeClass('g-dropzone-show')
             .html(`<i class="icon-docs"/> ${this._browseText}`);
-        this.files = e.originalEvent.dataTransfer.files;
+
+        var dataTransfer = e.originalEvent.dataTransfer;
+
+        // Require all dropped items to be files
+        if (!_.every(dataTransfer.items, _.bind(this._isFile, this))) {
```

Comment:
That doesn't work for me. How does it pass the item to the method?
How about:
`(item) => this._isFile(item)`